### PR TITLE
SCALRCORE-32442 Show who and when created/updated the variable

### PR DESCRIPTION
**Pull request by igor-melnyk-scalr**
### Changelog
* [x] I have updated CHANGELOG.md according to [CONTRIBUTING.md](../CONTRIBUTING.md)

### Documentation
* [ ] I have updated resource/data source documentation in [docs](../docs)

### State
* [ ] My changes affect the state
* [ ] I have included a state migration and a unit test for it

### CODE REVIEW

File: docs/data-sources/variable.md [hunk 1]
```diff
@@ -45,4 +45,16 @@ data "scalr_variable" "example2" {
 - `final` (Boolean) If the variable is configured as final. Indicates whether the variable can be overridden on a lower scope down the Scalr organizational model.
 - `hcl` (Boolean) If the variable is configured as a string of HCL code.
 - `sensitive` (Boolean) If the variable is configured as sensitive.
+- `updated_at` (String) Date/time the variable was updated.
+- `updated_by` (List of Object) Details of the user that updated the variable last time. (see [below for nested schema](#nestedatt--updated_by))
+- `updated_by_email` (String) Email of the user who updated the variable last time.
```
Comment (denkl) [inline docs/data-sources/variable.md:50]: @igor-melnyk-scalr do we really need to expose this field?

File: go.sum [hunk 2]
```diff
@@ -262,6 +262,8 @@ github.com/russross/blackfriday v1.6.0 h1:KqfZb0pUVN2lYqZUYRddxF4OR8ZMURnJIG5Y3V
 github.com/russross/blackfriday v1.6.0/go.mod h1:ti0ldHuxg49ri4ksnFxlkCfN+hvslNlmVHqNRXXJNAY=
 github.com/scalr/go-scalr v0.0.0-20240902160450-c71eed0dd7f4 h1:sPrrzxsDcHODrum/dgM931ux41BL+Ckn35av34yiGUw=
 github.com/scalr/go-scalr v0.0.0-20240902160450-c71eed0dd7f4/go.mod h1:p34SHb25YRvbgft7SUjSDYESeoQhWzAlxGXId/BbaSE=
+github.com/scalr/go-scalr v0.0.0-20240929210950-4e55784195ad h1:G+GZ3pm22r748k05/FBnBPKBmjG8xf7UMq1cEOkPG5Y=
```
Comment (denkl) [inline go.sum:265]: @igor-melnyk-scalr hashes of the previous versions must be removed

**Review by denkl — COMMENTED** [review]
@igor-melnyk-scalr don't forget to update CHANGELOG

File: docs/data-sources/variable.md (hunk 1 above)
Comment (igor-melnyk-scalr) [reply docs/data-sources/variable.md:50]: not sure, but provider must support it due to requirements in the ticket

File: go.sum (hunk 2 above)
Comment (igor-melnyk-scalr) [reply go.sum:263]: done

File: scalr/resource_scalr_variable.go [hunk 3]
```diff
@@ -231,6 +269,22 @@ func resourceScalrVariableRead(ctx context.Context, d *schema.ResourceData, meta
 	_ = d.Set("sensitive", variable.Sensitive)
 	_ = d.Set("description", variable.Description)
 	_ = d.Set("final", variable.Final)
+	_ = d.Set("updated_by_email", variable.UpdatedByEmail)
+
+	if variable.UpdatedAt != nil {
+		_ = d.Set("updated_at", variable.UpdatedAt.Format(time.RFC3339))
```
Comment (petroprotsakh) [inline scalr/resource_scalr_variable.go:275]: What is the reason to convert iso8601 to rfc3339?

**Review by petroprotsakh — APPROVED** [review]
1 question

File: scalr/resource_scalr_variable.go (hunk 3 above)
Comment (igor-melnyk-scalr) [reply scalr/resource_scalr_variable.go:275]: updated_at is a string in the resource schema, so the time.Time must be converted. in other case an error is raised

Comment (petroprotsakh) [reply scalr/resource_scalr_variable.go:275]: @igor-melnyk-scalr how is it different from this example? https://github.com/Scalr/terraform-provider-scalr/blob/a92bb6bda8a9f532a359ea1f7de6b6a783dbc087/scalr/resource_scalr_webhook.go#L260
Just want to make sure we work consistently with dates.

Comment (igor-melnyk-scalr) [reply scalr/resource_scalr_variable.go:275]: I saw this case, but I got error when I assign value without formatting

**Pull request merged**